terraform/modules/enclave/prometheus/main.tf

```diff
@@ -92,7 +93,7 @@ resource "aws_security_group_rule" "allow_prometheus_private" {
   security_group_id = "${aws_security_group.allow_prometheus.id}"
   type              = "ingress"
   from_port         = 9090
-  to_port           = 9090
+  to_port           = 9100
```

Comment:
So I believe this security group allows traffic from EC2 prometheus to the node exporter running on the EC2 instances. We could probably rename the security group rule from `allow_prometheus_private` to something that mentions `node_exporter` too.

We've changed the existing rule rather than adding a new one meaning that the previous rule is no longer needed. Could you explain why please as I wasn't able to figure it out myself. Thanks!

Reply:
We have modified the rule so that Prometheus can scrape node exporter on other instances. We have also change the name to make more clear as to the purpose of the rule.